examples/md-flexible/src/TimeDiscretization.cpp

```diff
@@ -20,9 +20,7 @@ void calculatePositions(autopas::AutoPas<ParticleType> &autoPasContainer,
   using autopas::utils::ArrayMath::dot;
   using autopas::utils::ArrayMath::mulScalar;
 
-  const auto maxAllowedDistanceMoved =
-      autoPasContainer.getVerletSkin() / (2 * autoPasContainer.getVerletRebuildFrequency());
-  const auto maxAllowedDistanceMovedSquared = maxAllowedDistanceMoved * maxAllowedDistanceMoved;
```

Comment:
Keep `maxAllowedDistanceMovedSquared`. There appears to be some error in the below code. On line 45 we compare `distanceMovedSquared` with `maxAllowedDistanceMoved` when it should be with `maxAllowedDistanceMovedSquared`

Reply:
Good catch. [9024538](https://github.com/AutoPas/AutoPas/pull/724/commits/9024538819583e7b1b486e58547f288071dcc64f)